<ui version="4.0" >
 <class>CobroPagoBase</class>
 <widget class="QWidget" name="CobroPagoBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Gesti√≥n de Cobros y Pagos</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../../bulmages.qrc" >:/Bulmafact48x48/images/png/48x48/i_bulmages_bancos_listado.48x48.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>5</number>
   </property>
   <property name="spacing" >
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../../../bulmages.qrc" >:/Bulmafact48x48/images/png/48x48/i_bulmages_bancos_listado.48x48.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame" >
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="mui_crear" >
          <property name="toolTip" >
           <string>Generar Asiento</string>
          </property>
          <property name="whatsThis" >
           <string>Generar Asiento</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../../bulmages.qrc" >:/Genericos32x32/images/png/i_fileNew.xpm</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_guardar" >
          <property name="toolTip" >
           <string>Actualizar</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../../bulmages.qrc" >:/Images/edit-save.svg</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_borrar" >
          <property name="toolTip" >
           <string>Actualizar</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../../bulmages.qrc" >:/Genericos32x32/images/png/i_dele.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_filtrar" >
          <property name="toolTip" >
           <string>Actualizar</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../../bulmages.qrc" >:/Genericos32x32/images/png/i_find.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_actualizar" >
          <property name="toolTip" >
           <string>Actualizar</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../../bulmages.qrc" >:/Genericos32x32/images/png/i_reload.png</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>571</width>
            <height>44</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame_2" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="textLabel1_3" >
            <property name="text" >
             <string>Cuenta</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="BusquedaCuenta" native="1" name="m_cuenta" />
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>141</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="textLabel1_2" >
            <property name="text" >
             <string>Procesado</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="m_procesado" >
            <item>
             <property name="text" >
              <string>TODO</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>PROCESADO</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>NO PROCESADO</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="textLabel1" >
            <property name="text" >
             <string>Tipo (Cobro / Pago)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="m_tipoprevcobro" >
            <item>
             <property name="text" >
              <string>TODO</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>COBROS</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>PAGOS</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="textLabel1_2_3" >
            <property name="text" >
             <string>Fecha Inicial</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="BusquedaFecha" native="1" name="m_firstDate" />
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>231</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="textLabel1_2_2_2" >
            <property name="text" >
             <string>Fecha Final</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="BusquedaFecha" native="1" name="m_lastDate" />
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="SubForm2Bc" name="mui_listado" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="textLabel1_4" >
       <property name="text" >
        <string>Total Cobros</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_totalCobros" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="font" >
        <font>
         <family>Sans Serif</family>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight</set>
       </property>
       <property name="readOnly" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="textLabel1_4_2" >
       <property name="text" >
        <string>Total Pagos</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_totalPagos" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="font" >
        <font>
         <family>Sans Serif</family>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight</set>
       </property>
       <property name="readOnly" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>BusquedaFecha</class>
   <extends>QWidget</extends>
   <header>../../../bulmalib/src/busquedafecha.h</header>
  </customwidget>
  <customwidget>
   <class>SubForm2Bc</class>
   <extends>QTableWidget</extends>
   <header>subform2bc.h</header>
  </customwidget>
  <customwidget>
   <class>BusquedaCuenta</class>
   <extends>QWidget</extends>
   <header>busquedacuenta.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../bulmages.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>mui_filtrar</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame_2</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>186</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel" >
     <x>189</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
